defaults/main.yml

```diff
@@ -105,10 +105,11 @@ cis_skip_lock_users:
   - sync
 
 # 5.4.4
-cis_umask_default: "027"
+#cis_umask_default: "027"
```

Comment:
Do we need this comment for something? And if not can we get rid of it?

Reply:
I'd just like to keep it for posterity in case I need to remember quickly what it was.